Analyzer/interface/Regions.h

```diff
@@ -0,0 +1,388 @@
+#ifndef SUSYBSMAnalysis_Analyzer_Regions_h
+#define SUSYBSMAnalysis_Analyzer_Regions_h
+
+#include "SUSYBSMAnalysis/Analyzer/interface/CommonFunction.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+
+#include <TCanvas.h>
+#include <TLegend.h>
+#include "TFile.h"
+#include "TDirectory.h"
+
+
+TRandom3* RNG = new TRandom3();
+
+float K = 1;
+float C = 1;
+
+// Scale the 1D-histogram given to the unit 
+void scale(TH1F* h){
+    h->Scale(1./h->Integral(0,h->GetNbinsX()+1));
+}
+
+// class using to definite signal and control regions. 
+class region{
```

Comment:
Class names should be upper case :P 
rule 2.6
https://cms-sw.github.io/cms_coding_rules.html

Reply:
Oooops... I'll fix it. (: